pytorch_lightning/utilities/cli.py

```diff
@@ -849,6 +862,19 @@ def _prepare_subcommand_kwargs(self, subcommand: str) -> Dict[str, Any]:
             fn_kwargs["datamodule"] = self.datamodule
         return fn_kwargs
 
+    def _set_seed(self) -> None:
+        """Sets the seed."""
+
+        config_seed = self._get(self.config, "seed_everything")
+
+        if isinstance(config_seed, bool):
+            if config_seed:
+                config_seed = _select_seed_randomly()
+
+        if config_seed is not False:
+            seed_everything(config_seed, workers=True)
+            self.config["seed_everything"] = config_seed
```

Comment:
This key might not exist if the user overrides `add_default_arguments_to_parser` and does not add the argument

Reply:
That could be. How do we deal with this then? 

if config_seed is not None and config_seed is not False should catch these cases.

However, do we also deal with the edge cases, where the user removes the attribute but sets seed_everything_default to True (the default value now, which could easily be forgotten) or some integer. Because otherwise, the user could accidentally think, that he seeds his program.

Reply:
> However, do we also deal with the edge cases, where the user removes the attribute but sets seed_everything_default to True (the default value now, which could easily be forgotten) or some integer.

I think this is fine, but we shouldn't add the key to the config if it didn't exist already IMO